config.yaml

```diff
@@ -0,0 +1,28 @@
+linkerdVersion: stable-2.8.1
+externalIssuer: false
+install:
+    skipTest: false
+    ha: false
+    # upgradeFromVersion: stable-2.7.0
+    flags:
+        - "--controller-log-level"
+        - "debug"
+        - "--proxy-log-level"
+        - "warn,linkerd2_proxy=debug"
+    addOns:
+        tracing:
+            enabled: true
+            collector:
+                resources:
+                    cpu:
+                        limit: 100m
+                        request: 10m
+                    memory:
+                        limit: 100Mi
+                        request: 50Mi
+    globalControlPlane:
```

Comment:
Hmm, the reason for choosing `skipTest` was that the zero value of booleans is `false`, which means, if the user chooses to skip the `enabled` field, it will always default to `false` and the test will not run unless they always manually specify `enabled: true`